Cargo.toml

```diff
@@ -27,12 +27,17 @@ incremental = false
 ethereum-consensus = { git = "https://github.com/ralexstokes/ethereum-consensus", rev = "cf3c404043230559660810bc0c9d6d5a8498d819" }
 beacon-api-client = { git = "https://github.com/ralexstokes/ethereum-consensus", rev = "cf3c404043230559660810bc0c9d6d5a8498d819" }
 
-reth = { git = "https://github.com/paradigmxyz/reth", rev = "5f82993c23164ce8ccdc7bf3ae5085205383a5c8" }
-reth-db = { git = "https://github.com/paradigmxyz/reth", rev = "5f82993c23164ce8ccdc7bf3ae5085205383a5c8" }
-reth-node-ethereum = { git = "https://github.com/paradigmxyz/reth", rev = "5f82993c23164ce8ccdc7bf3ae5085205383a5c8" }
-reth-basic-payload-builder = { git = "https://github.com/paradigmxyz/reth", rev = "5f82993c23164ce8ccdc7bf3ae5085205383a5c8" }
-alloy-signer = { git = "https://github.com/alloy-rs/alloy", rev = "77c1240" }
-alloy-signer-wallet = { git = "https://github.com/alloy-rs/alloy", rev = "77c1240" }
+reth = { git = "https://github.com/paradigmxyz/reth" }
```

Comment:
can we go ahead and specify `tag = "v1.0.8"`

otherwise the version can end up floating against the `Cargo.lock` as others work in this repo